CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+## [v0.1.2] - 2020-10-10
+
+### Added
+
+- Pin down `google-cloud-kms` to `1.4.0`
```

Comment:
since this is a changelog since `v0.1.1`, it would be appropriate to also include changes from https://github.com/tra-sg/discreetly/pull/8

The current changelog seems to refer to issues, but some of these changes aren't from an existing issue. I suggest this entry should go something like this

```
- Pin down `google-cloud-kms` to `1.4.0`, and update `pip cache` for tests ([#11])
- chore: Fix setup.py description and README.md release instructions ([#8])
```